install/kubernetes/tetragon/values.yaml

```
@@ -251,6 +251,20 @@ tetragon:
 tetragonOperator:
   # -- Enables the Tetragon Operator.
   enabled: true
+  # -- Number of replicas to run for the tetragon-operator deployment
+  replicas: 1
+  # -- Lease handling for an automated failover when running multiple replicas
+  failoverLease:
+    # -- Enable lease failover functionality
+    enabled: false
```

i don't mind running in the failover mode by default so that tetragon exercises this code path by default. i think this is what cilium does by default as well.

```suggestion
  # -- Number of replicas to run for the tetragon-operator deployment
  replicas: 2
  # -- Lease handling for an automated failover when running multiple replicas
  failoverLease:
    # -- Enable lease failover functionality
    enabled: true
```

we can release-note the helm value change here => https://github.com/cilium/tetragon/blob/main/contrib/upgrade-notes/latest.md?plain=1

---

Agree, Cilium does this as well. However, AFAICT, for Cilium the reliability/availability of the operator is way more important than for Tetragon. Hence, I'm not 100% if we should change the default. What do the others think?